_data/changelogs/component-tooltip.yml

```diff
@@ -5,6 +5,13 @@ type: component
 changelogURL:
 
 items:
+  - date: NNNN-NN-NN
+    summary: Updated component behavior to close active tooltips when the `escape` key is pressed.
+    summaryAdditional: This allows the component to meet the "dismissible" standard outlined in [WCAG 1.4.13](https://www.w3.org/WAI/WCAG21/Understanding/content-on-hover-or-focus.html).
```

Comment:
I was thinking we should update this link to match #2660 but it doesn't look like the [recommended page](https://www.w3.org/TR/WCAG21/#content-on-hover-or-focus) outlines the escape functionality the way this link does.

This is now making me second guess which one should be used. 🤔 

Let me know what you think!

Reply:
Using the link from #2661